LayoutTests/http/tests/resourceLoadStatistics/website-data-removal-for-site-navigated-to-with-unfiltered-link-decoration_before_short_deletion-expected.txt

```diff
@@ -0,0 +1,44 @@
+Check that script-writeable website data gets removed after a navigation with link decoration from a prevalent resource.
+
+Before deletion: Client-side cookie exists.
+Before deletion: Client-side persistent cookie exists.
+Before deletion: HttpOnly cookie exists.
+Before deletion: Regular server-side cookie exists.
+Before deletion: LocalStorage entry does exist.
+Before deletion: IDB entry does exist.
+Before deletion: Client-side cookie exists.
+Before deletion: Client-side persistent cookie exists.
+Before deletion: HttpOnly cookie exists.
+Before deletion: Regular server-side cookie exists.
+Before deletion: LocalStorage entry does exist.
+Before deletion: IDB entry does exist.
+
+operationalDays: 6
+Before deletion: Client-side cookie exists.
+Before deletion: Client-side persistent cookie exists.
+Before deletion: HttpOnly cookie exists.
+Before deletion: Regular server-side cookie exists.
+Before deletion: LocalStorage entry does exist.
+Before deletion: IDB entry does exist.
```

Comment:
Description does not match result, and it seems a bit weird to check data exists "Before deletion". Maybe this test can be merged with the one with 7-day check, and the log can start with "Results:" or "After X operational days:".

Reply:
I want to check "before deletion" because this allows us to dump and verify the load statistics database state before deletion happens. The current implementation only allows dumping state at the end of the test, and that is why I separated the tests.